dotnet/src/support/WebDriver.Support.csproj

```diff
@@ -25,51 +21,8 @@
     <PackageId>Selenium.Support</PackageId>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <FileVersion>4.0.0.0</FileVersion>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net45|AnyCPU'">
-    <DocumentationFile>..\..\..\build\cli\Release\net45\WebDriver.Support.xml</DocumentationFile>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
```

Comment:
Was this config used to generate the code documentation and since we do it different now this is not needed anymore?

Reply:
It is replaced by `<GenerateDocumentationFile>true</GenerateDocumentationFile>`

Reply:
Thanks for the explanation.